### PR TITLE
Add timm (aka pytorch-image-models)

### PR DESCRIPTION
**Pull request by ndmaxar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ndmaxar** [comment]
@rluria14 and @oblute ok to be maintainers?

**Comment by ndmaxar** [comment]
@conda-forge/staged-recipes this recipe is ready for review.

File: recipes/timm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b7a7061885bd2797603fb6482f2337be9e6ba4199edac624a8b27aa5612fca77
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
```
Comment (scopatz) [inline recipes/timm/meta.yaml:15]: Please remove this and make this noarch

File: recipes/timm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  host:
+    - pip
+    - python ==3.7
```
Comment (scopatz) [inline recipes/timm/meta.yaml:20]: Is this really 3.7 only, and not >=3.7?

File: recipes/timm/meta.yaml (hunk 1 above)
Comment (ndmaxar) [reply recipes/timm/meta.yaml]: The windows build fails consistently.

File: recipes/timm/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b7a7061885bd2797603fb6482f2337be9e6ba4199edac624a8b27aa5612fca77
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+  # skip: True  # [win]
```
Comment (scopatz) [inline recipes/timm/meta.yaml:16]: Please remove comment line

File: recipes/timm/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/timm/meta.yaml]: That's OK

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**